app/src/main/java/com/github/sdp/tarjetakuna/utils/TemporaryCards.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.sdp.tarjetakuna.utils
+
+import com.github.sdp.tarjetakuna.model.*
+
+/**
+ * Represents temporary cards that are generated with a loop
+ */
+object TemporaryCards {
+    fun generateCards(): ArrayList<MagicCard> {
+        val cardsArray = ArrayList<MagicCard>()
+        for (i in 0..39) {
+            val number = if (i + 1 < 10) {
+                "0${i + 1}"
+            } else {
+                "${i + 1}"
+            }
+            val name = "Magic 19$number"
+            val card = MagicCard(
+                "Ambush Paratrooper $number",
+                "Flash, Flying\n5: Creatures you control get +1/+1 until end of turn.",
+                MagicLayout.NORMAL,
+                2,
+                "{1}{W}",
+                MagicSet("BRO", name),
+                i + 1,
+                "https://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=149935&type=card",

```

Comment:
Please use for the moment data from Scryfall API and not Gatherer.

Reply:
Would you be able to give me an example of the link please, I do not manage to find them on the main branch

Reply:
You have the API documentation [here](https://scryfall.com/docs/api).

If you click on this link https://api.scryfall.com/cards/random?pretty=true, it is the data of a random card. image_uris contains images, for example this one for the normal one : https://cards.scryfall.io/normal/front/8/7/87352716-4cf6-4b2f-bb0a-b7aafae64478.jpg?1562650023